src/snapred/backend/dao/state/CalibrantSample/Geometry.py

```diff
@@ -1,26 +1,26 @@
-from typing import Optional
+from typing import List, Optional
 
 from pydantic import BaseModel, root_validator
 
 
 class Geometry(BaseModel):
     """Class to hold Geometry data for Calibrant Samples
-    form: string must be 'cylinder' or 'sphere'
-    radius: single float (cm)
-    illuminated height: single float (cm)
-    total height: single float (cm) (not used for sphere)"""
+    Shape: string must be 'Cylinder' or 'Sphere'
```

Comment:
I would prefer that member variables start with a lowercase letter, unless this is somehow required to be in lockstep with mantid

Reply:
So did some testing and it looks like those are not case sensitive to mantid, will update to be lowercase